data/blood/glucose/glucose.go

```diff
@@ -60,3 +61,49 @@ func NormalizeValueForUnits(value *float64, units *string) *float64 {
 	}
 	return value
 }
+
+const (
+	// MgdL_Per_MmolL renames MmolLToMgdLConversionFactor in an alternate way.
+	MgdL_Per_MmolL = MmolLToMgdLConversionFactor
+	// MmolL_Per_MgdL inverts the MmolL_Per_MgdL conversion factor.
+	MmolL_Per_MgdL = 1 / MgdL_Per_MmolL
+)
+
+// Convert between common blood glucose measurement units.
+//
+// Results are rounded to five decimal places. However, if fromUnits and
+// toUnits are the same, the value is returned unchanged. If either fromUnits
+// or toUnits aren't handled, Convert panics.
+//
+// Convert is modeled to return results as similar as possible to
+// NormalizeValueForUnits, but without the hassle of pointer values, and with
+// the ability to convert between arbitrary units (as supported).
+func Convert(value float64, fromUnits, toUnits string) float64 {
+	from, to := normalizeUnitsCase(fromUnits), normalizeUnitsCase(toUnits)
+	if from == to {
+		return value
+	}
+	if from == MgdL && to == MmolL {
+		return Round5(value * MmolL_Per_MgdL)
+	}
+	if from == MmolL && to == MgdL {
+		return Round5(value * MgdL_Per_MmolL)
+	}
+	panic(fmt.Errorf("unhandled conversion %q => %q", fromUnits, toUnits))
```

Comment:
Please do not use `panic` within the code. (Outside of test code it is used in only a very, very few specific uses cases.) Return an error and, if possible and appropriate, log.

Reply:
This was poor choice on my part, you're right.